WooCommerce/Classes/Authentication/AuthenticatedWebViewController.swift

```diff
@@ -193,17 +210,24 @@ extension AuthenticatedWebViewController: WKNavigationDelegate {
 
     func webView(_ webView: WKWebView, didStartProvisionalNavigation navigation: WKNavigation!) {
         progressBar.setProgress(0, animated: false)
+        activityIndicator.startAnimating()
     }
 
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        activityIndicator.stopAnimating()
         guard let url = webView.url else {
             return
         }
         viewModel.didFinishNavigation(for: url)
     }
 
+    func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: any Error) {
+        activityIndicator.stopAnimating()
+    }
+
```

Comment:
nice work adding handling for this case and `didFailProvisionalNavigation` below.